src/solver/conv_hip_implicit_gemm_3d_grouped_bwd_xdlops.cpp

```diff
@@ -292,15 +302,18 @@ struct CKArgs
     }
 
     template <typename ConvPtr>
-    auto MakeArgPtr(const ConvPtr& conv_ptr, const ConvDataTensors& tensors) const
+    auto MakeArgPtr(const ConvPtr& conv_ptr,
+                    const ConvDataTensors& tensors,
+                    float alpha,
+                    float beta) const
     {
-        return MakeArgPtr(conv_ptr, tensors.out, tensors.w, tensors.in);
+        return MakeArgPtr(conv_ptr, tensors.out, tensors.w, tensors.in, alpha, beta);
     }
 
     template <typename ConvPtr>
     bool IsSupportedBy(const ConvPtr& conv_ptr) const
     {
-        auto arg_ptr = MakeArgPtr(conv_ptr, nullptr, nullptr, nullptr);
+        auto arg_ptr = MakeArgPtr(conv_ptr, nullptr, nullptr, nullptr, 1.0f, 0.0f);
```

Comment:
suggestion: perhaps we should define constants for `CONV_DEFAULT_ALPHA` and `CONV_DEFAULT_BETA` as you are using `1.0` and `0.0` in other places like the constructor for `ProblemDescription` and `*InvokeParams`.